Algebraic Topology/algebraic-topology.tex

```diff
@@ -156,7 +156,7 @@ \section{Homotopy}
 \mydate{d2}{20}{1}{2017}
 
 \begin{defn}
-  A \de{category} is a directed graph.  Each vertex is called an
+  A \de{category} is a directed graph. Each vertex is called an %
```

Comment:
See here: https://en.wikipedia.org/wiki/Category_(mathematics)
Obviously there are other correct definitions. What is stated here is not incorrect, it is just incomplete. In other words, a category is a directed graph as described, but it has additional structure. It seems like a lot to go into the formal definition of a category, though.